src/pages/Home/Content/Section/ItemList/ItemCard/ItemCard.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { FoodItem } from "../../../../../../DummyData/DataTypes";
+import { ItemCardStyle } from "./ItemCardStyle";
+
+type Props = {
+  item: FoodItem;
+};
+
+export default function ItemCard({ item }: Props) {
+  return (
+    <ItemCardStyle>
+      <img src={item.img} alt={item.description} />
+      <h5>{item.name}</h5>
+      <h5>{item.basePrice}</h5>
+      {/*BUG: number 8.60 converted to 8.6 Should use string instead.*/}
```

Comment:
Price should always be in 2dp not 1dp or whole number

Reply:
You'll probably need a util function to convert the numeric price data type into an appropriate string